content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -22,6 +22,100 @@ Configuring the [aggregation layer](/docs/concepts/extend-kubernetes/api-extensi
 There are a few setup requirements for getting the aggregation layer working in your environment to support mutual TLS auth between the proxy and extension apiservers. Kubernetes and the kube-apiserver have multiple CAs, so make sure that the proxy is signed by the aggregation layer CA and not by something else, like the master CA.
 {{< /note >}}
 
+{{% /capture %}}
+
+{{% capture authflow %}}
+
+## Authentication Flow
+
+Unlike Custom Resource Definitions (CRDs), the Aggregation API involves another server - your Extension API Server - in addition to the standard Kubernetes apiserver. The Kubernetes apiserver will need to communicate with your extension server, and your extension server will need to communicate with the Kubernetes apiserver. In order for this communication to be secured, the apiserver uses x509 certificates to authenticate itself to the extension server.
```

Comment:
nit: suggest `API server` over `API Server` and `apiserver`

Reply:
also suggest explicitly saying "Kubernetes API server" and "Extension API server" everywhere instead of "apiserver" for clarity

Reply:
Yeah, that makes sense. Elsewhere in the docs I have seen "apiserver" so stuck with it. I don't care much one way or the other though, so will follow your suggestion.

Reply:
Actually, the original document uses "apiserver" everywhere (7 times). You sure you want me to change all of those as well?

Reply:
But adding explicit "Kubernetes apiserver" vs "extension apiserver" (or "API server") is a definite.

Reply:
Updated to use "Kubernetes apiserver" and "extension apiserver" everywhere. If you feel we should change all cases of "apiserver" to "API server" (including the pre-existing ones), no problem, quick search-and-replace.